Source/santad/EventProviders/EndpointSecurityTestUtil.h

```diff
@@ -0,0 +1,50 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+
+#include <EndpointSecurity/EndpointSecurity.h>
+#include <bsm/libbsm.h>
+
+es_string_token_t MakeStringToken(const NSString *s);
+
+@interface ESResponse : NSObject
+@property(nonatomic) es_auth_result_t result;
+@property(nonatomic) bool shouldCache;
+@end
+
+// Singleton wrapper around all of the kernel-level EndpointSecurity framework functions.
+@interface MockEndpointSecurity : NSObject
+- (void)reset;
+- (void)registerResponseCallback:(void (^)(ESResponse *))callback;
```

Comment:
As the block type is referenced in a few other places (the array inside the .mm, for instance) it may be worth making a typedef for it:

`typedef void (^ESCallback)(ESResponse *);`

And then you can change this to:

`- (void)registerResponseCallback:(ESCallback)callback;`